searx/settings_defaults.py

```diff
@@ -142,6 +142,7 @@ def apply_schema(settings, schema, path_list):
         'debug': SettingsValue(bool, False, 'SEARXNG_DEBUG'),
         'instance_name': SettingsValue(str, 'SearXNG'),
         'contact_url': SettingsValue((None, False, str), None),
+        'enable_metrics': SettingsValue(bool, True),
```

Comment:
I miss a documentation in the [admin/engines/settings/general](https://docs.searxng.org/admin/engines/settings.html#general) section.

Reply:
Done